benchmarks/src/sha3-rocc.c

```diff
@@ -43,12 +51,15 @@ int main() {
     for(i = 0; i < SHA3_256_DIGEST_SIZE; i++){
       printf("output[%d]:%d ==? results[%d]:%d \n",i,output[i],i,result[i]);
       if(output[i] != result[i]) {
-        printf("Outputs don't match!\n");
+        printf("Failed: Outputs don't match!\n");
+        printf("SHA execution took %lu cycles\n", total_cycles);
```

Comment:
It doesn't matter now, but if there are multiple tests, the cycle count for an individual test is `end - start`, not `total_cycles`.

Reply:
Sorry, I don't follow.

Reply:
Right now, it's just a matter of semantics.  Suppose another test is inserted before this one.  `total_cycles` would include the time spent in that preceding test instead of just this one.

Reply:
Sure. I didn't assume that... but I don't think that is a problem yet.

Reply:
I don't see why it couldn't be `total_cycles = end - start` for simplicity, but it isn't a big deal.

Reply:
Oh wait. I realize what I did. I just saw the while loop and thought it would run multiple times... didn't realize its a while(0). Ill fix it.